src/components/Board/components/Column/index.js

```diff
@@ -1,5 +1,7 @@
+import ReactDOM from 'react-dom'
```

Comment:
great that it worked, please add links to sources you used like probably this one:
https://github.com/atlassian/react-beautiful-dnd/blob/master/stories/src/virtual/react-virtualized/list.jsx